### PR TITLE
Erin/casenote card view

### PR DESCRIPTION
**Pull request by erinysong**
## FEATURE TITLE

Case Note Card Full View
Added seed data for casenotes

### Related PRs

Optional - any related PRs you're waiting on, or PRs that will conflict, etc

### Migrations

Optional - if you added anything to the database through migration(s)

### Tests Performed, Edge Cases

Lol we'll see if we test

### Screenshots

Add screenshots!!! If you'd like
image
image

CC: @adowski

### CODE REVIEW

File: app/javascript/components/CaseNoteCardModal.jsx [hunk 1]
```diff
@@ -0,0 +1,162 @@
...
+                    </div>
+                    </div>
+                    
+                </Dialog>
+
+            </div>
+        );
+    }
+}
+
```
Comment (julianrkung) [inline app/javascript/components/CaseNoteCardModal.jsx]: Don't forget to add `propTypes` for modal here!

File: app/javascript/components/CaseNoteCardModal.jsx [hunk 2]
```diff
@@ -0,0 +1,162 @@
...
+    casenoteDescStyle: {
+        marginLeft: '20px',
+        paddingTop: '20px',
+    },
+    dialogActionsStyle: {
+        padding: '30px',
+    },
+    MUIRichTextEditorStyle: {
+        border: '5px solid',
+        padding: '10px',
```
Comment (julianrkung) [inline app/javascript/components/CaseNoteCardModal.jsx]: The styles related to MUIRichTextEditor show up in a lot of different files. Would be more manageable to wrap the MUIRichTextEditor in its own component or centralize it so if we decide to change some style we only have to change it in one place

Comment (julianrkung) [reply app/javascript/components/CaseNoteCardModal.jsx]: ^ note goes for most repeated styles also. the textEditor was just the one i noticed

**Comment by julianrkung** [comment]
CaseNotes overflow when there's too much text also. Probably requires a change to `casenoteDescStyle` in `CaseNoteCard` but I don't know what proper functionality is so won't make the change
